packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -465,24 +467,27 @@ function AsyncLoadingExample() {
   });
 
   return (
-    <Table width={1000} height={500} isQuiet sortDescriptor={list.sortDescriptor} onSortChange={list.sort}>
-      <TableHeader>
-        <Column uniqueKey="score" width={100} allowsSorting>Score</Column>
-        <Column uniqueKey="title" isRowHeader allowsSorting>Title</Column>
-        <Column uniqueKey="author" width={200} allowsSorting>Author</Column>
-        <Column uniqueKey="num_comments" width={100} allowsSorting>Comments</Column>
-      </TableHeader>
-      <TableBody items={list.items} isLoading={list.isLoading} onLoadMore={list.loadMore}>
-        {item =>
-          (<Row uniqueKey={item.data.id}>
-            {key => 
-              key === 'title'
-                ? <Cell textValue={item.data.title}><Link isQuiet><a href={item.data.url} target="_blank">{item.data.title}</a></Link></Cell>
-                : <Cell>{item.data[key]}</Cell>
-            }
-          </Row>)
-        }
-      </TableBody>
-    </Table>
+    <div>
+      <ActionButton onPress={() => list.remove(list.items[0].data.id)}>Remove first item</ActionButton>
```

Comment:
Added a button to delete first item in async list to test that loadMore animations are gone but deletion animation is still there. Pretty barebones example but can update to be more like the CRUD example later